Python/Product/TestAdapter.Executor/TestExecutor.cs

```diff
@@ -877,10 +892,12 @@ enum PythonDebugMode {
             PythonAndNative
         }
 
-        private static string PtvsdSearchPath {
-            get {
+        private static string GetDebuggerPath(bool useLegacyDebugger) {
```

Comment:
Please keep the "Search" in the name "GetDebuggerSearchPath", because this isn't returning the path to the debugger folder, it's returning the search path for it (ie its parent folder).